### ISSUE TITLE
Python project names should not include '-' character

### ISSUE THREAD

**Issue by alan-copeland-keysight** [opentap/OpenTap.Python#112]
Causes a problem with the generated .NET namespace

**Comment by rmadsen-ks** [comment]
Hi @alan-copeland-keysight, I believe you, but in order to better understand the issue, can you post some of the error messages you got when trying to do this?

**Comment by alan-copeland-keysight** [comment]
For example, use the --project-name ```bad-name```:
```tap python new-project --project-name bad-name --directory bad-name```

Try to build using ```dotnet build```:
```
MSBuild version 17.3.2+561848881 for .NET
  Determining projects to restore...
  Restored c:\src\bad-name\bad-name.Api\bad-name.Api.csproj (in 4.15 sec).
c:\src\bad-name\bad-name.Api\bad-name.Api.csproj : warning OpenTAP Install: Installer : Package 'Python' contains possi
bly relevant plugins for next package installations, but these will not be loaded.
c:\src\bad-name\bad-name.Api\ExampleApi.cs(6,20): error CS0116: A namespace cannot directly contain members such as fie
lds, methods or statements [c:\src\bad-name\bad-name.Api\bad-name.Api.csproj]
c:\src\bad-name\bad-name.Api\ExampleApi.cs(6,14): error CS1514: { expected [c:\src\bad-name\bad-name.Api\bad-name.Api.c
sproj]
c:\src\bad-name\bad-name.Api\ExampleApi.cs(7,1): error CS1022: Type or namespace definition, or end-of-file expected [c
:\src\bad-name\bad-name.Api\bad-name.Api.csproj]

Build FAILED.

c:\src\bad-name\bad-name.Api\bad-name.Api.csproj : warning OpenTAP Install: Installer : Package 'Python' contains possi
bly relevant plugins for next package installations, but these will not be loaded.
c:\src\bad-name\bad-name.Api\ExampleApi.cs(6,20): error CS0116: A namespace cannot directly contain members such as fie
lds, methods or statements [c:\src\bad-name\bad-name.Api\bad-name.Api.csproj]
c:\src\bad-name\bad-name.Api\ExampleApi.cs(6,14): error CS1514: { expected [c:\src\bad-name\bad-name.Api\bad-name.Api.c
sproj]
c:\src\bad-name\bad-name.Api\ExampleApi.cs(7,1): error CS1022: Type or namespace definition, or end-of-file expected [c
:\src\bad-name\bad-name.Api\bad-name.Api.csproj]
    1 Warning(s)
    3 Error(s)

Time Elapsed 00:00:14.04
```

From ExampleApi.cs:
![image](https://github.com/opentap/OpenTap.Python/assets/69871845/66b60502-b035-4d21-874f-d6ccf9c1c539)

**Comment by alan-copeland-keysight** [comment]
Seen with Python plugin 3.0.1

**Comment by rmadsen-ks** [comment]
Ok, thanks. This makes sense.

**Comment by rmadsen-ks** [comment]
Fixed by #166
